cves/kernel/CVE-2020-12770.yml

```diff
@@ -358,8 +385,11 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
```

Comment:
Make sure to change fix in this first sentence to fixed. The second sentence is also a little unclear to me. Maybe mention what comparison was changed and what the forgotten function would've done.

Reply:
The second sentence does feel a little disjointed. I will fix it to sound better.